static/src/javascripts/projects/commercial/modules/dfp/queue-advert.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import dfpEnv from 'commercial/modules/dfp/dfp-env';
+
+const queueAdvert = (advert: Object) => {
+    dfpEnv.advertsToLoad.push(advert);
+    // Add to the array of ads to be refreshed (when the breakpoint changes)
+    // only if its `data-refresh` attribute isn't set to false.
+    if (advert.node.getAttribute('data-refresh') !== 'false') {
+        dfpEnv.advertsToRefresh.push(advert);
+    }
+};
+
+export default queueAdvert;
```

Comment:
Would it be possible to use `export { queueAdvert }` instead?

We are [trying to use this pattern](https://github.com/guardian/frontend/wiki/So-you-want-to-ES6%3F#-code-style) instead of `export default`, as it makes the code more greppable.